src/components/TodoListItem.js

```diff
@@ -17,4 +18,8 @@ const TodoListItem = (props) => {
   );
 };
 
+TodoListItem.propTypes = {
+  handleRemoveClick: PropTypes.func,
```

Comment:
Also here, `handleRemoveClick` is a function you are defining within the component, not one of its props. The props for this component are destructured on line 6, go ahead and add their correct `propTypes`.
```suggestion
```